projects/opus/opus_multi_fuzzer.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
```

Comment:
this is not recommended to do, please add a size check in the fuzz target source `if (size > kMaxSize) return 0;`

Reply:
I was under the impression that this option had the ability to increase the maximum `size` of inputs generated, as indicated by the following message when running libFuzzer locally: `-max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes`. Also, while adding this, I was referencing the .options file for the existing fuzzer, located [here](https://gitlab.xiph.org/xiph/opus/-/blob/master/tests/opus_decode_fuzzer.options). Is that incorrect as well?

Reply:
ClusterFuzz/OSS-Fuzz automatically tries different length arguments by explicitly providing -max_len. This is not needed at all. Can you remove this file altogether, i think your intention was to try big file lengths which we already do via fuzzing strategy.

Reply:
I see, thanks for clarifying. I'll remove this in the next push.